TLSharp.Core/Network/MtProtoSender.cs

```diff
@@ -167,24 +203,26 @@ public async Task SendPingAsync(CancellationToken token = default(CancellationTo
             await Receive(pingRequest, token).ConfigureAwait(false);
         }
 
-        private bool processMessage(ulong messageId, int sequence, BinaryReader messageReader, TLMethod request, CancellationToken token = default(CancellationToken))
+        private async Task<bool> processMessageAsync(ulong messageId, int sequence, BinaryReader messageReader, TeleSharp.TL.TLMethod request, CancellationToken token = default(CancellationToken))
```

Comment:
now that you're changing this line, let's use proper notation: PascalCase instead of camelCase